server/notification-providers/cachet.js

```diff
@@ -0,0 +1,50 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class Cachet extends NotificationProvider {
+
+    name = "Cachet";
+
+    _bodyFactory = (status, cachetFatalState) => {
+        if (status === DOWN) {
+            return `{"status":"`+cachetFatalState+`"}`;
+        } else if (status === UP) {
+            return `{"status":"1"}`;
+        } else {
+            return `{"status":"O"}`;
+        }
+    };
+
+    _sendNotification = async (url, token, body) => {
+        await axios.put(
+            url,
+            body,
+            {
+                headers: {
+                    'Content-Type': 'application/json',
+                    'X-Cachet-Token': token,
+                }
+            }
+        );
+    };
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            if (heartbeatJSON == null) {
+                return "Sent Successfully.";
```

Comment:
Is it worth sending test data here instead of just returning. For example see https://github.com/louislam/uptime-kuma/blob/92ab2b12d0e62897e3f0f8e73c161eac75302450/server/notification-providers/feishu.js#L13-L21